imod/mf6/regrid/regrid_schemes.py

```diff
@@ -1,24 +1,10 @@
-from typing import ClassVar, Protocol, Tuple, TypeAlias
+from typing import Tuple, TypeAlias
 
-from pydantic import ConfigDict
 from pydantic.dataclasses import dataclass
 
-from imod.mf6.utilities.regridding_types import RegridderType
+from imod.util.regrid_method_type import _CONFIG, RegridderType, RegridMethodType
 
 _RegridVarType: TypeAlias = Tuple[RegridderType, str] | Tuple[RegridderType]
```

Comment:
You already defined this TypeAlias in ``imod/util/regrid_method_type.py``, you can use that one instead.

Reply:
done